e2e/api/steps.js

```diff
@@ -724,7 +724,13 @@ const assertValidData = async (data, pageId, solicitor) => {
     caseData = removeUiFields(pageId, caseData);
   }
 
-  assert.deepEqual(responseBody.data, caseData);
+  try {
+    assert.deepEqual(responseBody.data, caseData);
+  } 
+  catch(err) {
+    console.log('Valid data is failed with mismatch ..', err);
```

Comment:
I would suggest something like the following:
"Data is failing due to a mismatch. Please make sure that the corresponding civil-ccd-definition entries are correct and that civil-service handles them correctly. Before re-launching the build try clearing the pods to ensure the latest master from civil-ccd-definition and civil-service are fetched. If the issue persists, check that the code in your PR is correct and check the logs in the pods themselves using 'kubectl logs' or an application like Lens"

More verbose, but would start giving some guidance on how to solve the issue.

Reply:
```suggestion
    console.info("Data is failing due to a mismatch. Make sure that the corresponding civil-ccd-definition entries are correct and that civil-service handles them correctly. Before re-launching the build try clearing the pods to ensure the latest master from civil-ccd-definition and civil-service are fetched. If the issue persists, check that the code in your PR is correct and check the logs in the pods themselves using 'kubectl logs' or an application like Lens");
    console.error('Valid data is failed with mismatch ..', err);
```

Two reasons for this:
1. we give an error that also tells the developers possible paths to the solution, especially since we find ourselves in a situation where people don't know how to react to the errors
2. since it's an error, your line should probably log on on `console.error`

Reply:
Moved console.log to error. But other message does not seem to be needed here. As we are already adding two logs
console.log(`asserting page: ${pageId} has valid data`);
    console.error('Validate data is failed due to a mismatch ..', err);
So, both should be sufficient for anyone to debug the error.

Reply:
Alright, let's give it a shot. Resolving and approving.